test/functional/forall/test-forall-openmp.cpp

```diff
@@ -0,0 +1,22 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "test-forall-rangesegment.hpp"
+
+// Generate OMP Type List
+#if defined(RAJA_ENABLE_OPENMP)
+using OMPTypes = list<RAJA::omp_parallel_exec<RAJA::seq_exec>,
```

Comment:
@mdavis36 what is this policy? Does it create an openmp parallel region and then do a sequential loop inside it?  If so, I didn't know we had that.

Please also add RAJA::omp_parallel_for_exec policy to list.

Reply:
That is exactly what it does. Added.